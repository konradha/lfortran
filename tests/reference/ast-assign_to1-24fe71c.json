{
    "basename": "ast-assign_to1-24fe71c",
    "cmd": "lfortran --indent --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/assign_to1.f90",
    "infile_hash": "0da8cafe77fcfae28448ae5b873c873d6e643054caf9f348813f834b",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-assign_to1-24fe71c.stdout",
    "stdout_hash": "cda327a60feaf999c4840be47745217e4eb099cbb091343bf1e98cd9",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}