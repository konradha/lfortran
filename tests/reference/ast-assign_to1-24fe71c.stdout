(TranslationUnit 
    [(Program 
        main 
        () 
        [] 
        [] 
        [] 
        [(If 
            1000 
            () 
            (< 0 1) 
            [(Assign 
                0 
                1 
                k 
                ()
            )] 
            [] 
            () 
            () 
            ()
        )
        (GoTo 
            100 
            k 
            () 
            [] 
            ()
        )
        (Print 
            1 
            () 
            [(String 
                "run"
            )] 
            ()
        )] 
        []
    )]
)
